src/fightPage/resultModalPopUpFunctions.ts

```diff
@@ -0,0 +1,22 @@
+import { GameHandler } from "../gameHandlerClass";
+import { createFightPage } from "./renderFightPage";
+import { Player } from "../playerClass";
+
+export const showResultModal = (
```

Comment:
I think that showResultModal should take as an argument gameHandler in order to retrieve the winner's name and display it in "modal-header"

Reply:
I added gameHandle. You are right :) it will be usefull to get winner name. After that it will be set to 'modal-header'. 

I think functions can be finnished when MNG-114 will be done. Now it is hard to implement it, when I can't check how it works. What do you think?

Reply:
You could set the name to GameHandler.getWinner() and comment it out - that's how we will get the name of the player who wins. But if you'd prefer to do it later, I think it's ok ;)